.kokoro/build.sh

```diff
@@ -112,6 +112,25 @@ clirr)
     mvn -B -ntp -Denforcer.skip=true clirr:check
     RETURN_CODE=$?
     ;;
+conformance)
+    # Build and start the proxy in a separate process
+    pushd .
+    cd test-proxy
+    mvn clean install
+    nohup java -Dport=9999 -jar target/google-cloud-bigtable-test-proxy-0.0.1-SNAPSHOT.jar &
+    proxyPID=$!
+    popd
+
+    # Invoke the test
+    pushd .
+    cd cloud-bigtable-clients-test/tests
```

Comment:
Where is this `cloud-bigtable-clients-test`  directory?  I don't see it under test-proxy?

Reply:
It's checked out in the ci.yaml:
- uses: actions/checkout@v3
      with:
        repository: googleapis/cloud-bigtable-clients-test
        ref: main
        path: cloud-bigtable-clients-test

The end result is like this:

java-bigtable |
                        |--- test_proxy
                        |--- cloud-bigtable-clients-test

When the if branch is being executed, the current dir "." is at the root (i.e. java-bigtable/)